mission_control/navigator_missions/nodes/move_command

```diff
@@ -22,7 +22,7 @@ def main(args):
 
     action_kwargs = {'move_type': args.movetype, 'speed_factor': args.speedfactor}
 
-    if args.blind is not None:
+    if args.blind not in  (None, False):
```

Comment:
Don't you want to pass blind in as False if the user specifies? (Even tho that's default)